### PR TITLE
Added usage of subprocess for sending events

### PR DESCRIPTION
**Pull request by popovaan**
Added usage of subprocess for sending events
Fixes https://github.com/openvinotoolkit/openvino/issues/27562

Ticket: CVS-157449

### CODE REVIEW

File: src/backend/backend_ga4.py [hunk 1]
```diff
@@ -36,15 +38,26 @@ def send(self, message: dict):
...
                 return
+            process = multiprocessing.Process(target=_send_func, args=(req,))
+            process.daemon = True
+            process.start()
+
+            process.join(self.timeout)
+            if process.is_alive():
+                process.terminate()
 
-            request.urlopen(req)  # nosec
```
Comment (rkazants) [inline src/backend/backend_ga4.py:47]: why can't we use timeout arg for `urlopen`? This way we don't need to create a new process.

File: src/backend/backend_ga4.py [hunk 2]
```diff
@@ -11,12 +11,14 @@
...
 from ..utils.cid import get_or_generate_cid, remove_cid_file
 from ..utils.params import telemetry_params
+import multiprocessing
 
 
 class GA4Backend(TelemetryBackend):
     id = 'ga4'
     cid_filename = 'openvino_ga_cid'
     old_cid_filename = 'openvino_ga_uid'
+    timeout = 1.0
```
Comment (rkazants) [inline src/backend/backend_ga4.py:21]: I think it makes sense to increase upto 3 sec.

File: src/backend/backend_ga4.py (hunk 1 above)
Comment (popovaan) [reply src/backend/backend_ga4.py:47]: Timeout in `urlopen` just doesn't work. I tried to use it, but `request.urlopen(req)` still hangs for about 15-20 sec. Probably this timeout works for file downloading, but I'm not sure.

File: src/backend/backend_ga4.py (hunk 2 above)
Comment (popovaan) [reply src/backend/backend_ga4.py]: Done.

**Review by rkazants — APPROVED** [review]

**Pull request merged**